torch/csrc/jit/python/init.cpp

```diff
@@ -667,6 +667,21 @@ void initJITBindings(PyObject* module) {
           [](std::string loggingOption) -> void {
             ::torch::jit::set_jit_logging_levels(loggingOption);
           })
+      .def(
+          "_jit_set_logging_stream",
+          [](std::string stream_name) -> void {
+            if(stream_name == "stdout") {
+              ::torch::jit::set_jit_logging_output_stream(std::cout);
+            }
+            else if(stream_name == "stderr")  {
+              ::torch::jit::set_jit_logging_output_stream(std::cerr);
+            }
+            else {
```

Comment:
Do you mind running clang-format on the PR? You can use `tools/linter/clang_format_all.py` - it downloads a proper version of clang-format and runs it on the entire codebase (if it reformats files not touched by my changes, I usually undo those reformats)